httpdocs/ajax/server/list.php

```diff
@@ -12,7 +12,9 @@ function error($message) {
   $mysqli->set_charset('utf8');
   $offset = isset($data->offset) ? intval($data->offset) : 0;
   $limit = isset($data->limit) ? intval($data->limit) : 10;
-  $query = "SELECT * FROM server ORDER BY `share` - `load` DESC LIMIT $limit OFFSET $offset";
+  $branch = basename(dirname(__FILE__, 4));
+  $join= "JOIN server_branch ON server.id=server_branch.id WHERE branch=\"$branch\"";
+  $query = "SELECT * FROM server $join ORDER BY `share` - `load` DESC LIMIT $limit OFFSET $offset";
```

Comment:
Technically, it's not just a join, but also includes a WHERE condition...
```suggestion
  $join_and_condition = "JOIN server_branch ON server.id=server_branch.id WHERE branch=\"$branch\"";
  $query = "SELECT * FROM server $join_and_condition ORDER BY `share` - `load` DESC LIMIT $limit OFFSET $offset";
```